pkg/volume/gcepd/attacher.go

```diff
@@ -141,6 +141,47 @@ func (attacher *gcePersistentDiskAttacher) VolumesAreAttached(specs []*volume.Sp
 	return volumesAttachedCheck, nil
 }
 
+func (attacher *gcePersistentDiskAttacher) BulkVerifyVolumes(volumesByNode map[types.NodeName][]*volume.Spec) (map[types.NodeName]map[*volume.Spec]bool, error) {
+	volumesAttachedCheck := make(map[types.NodeName]map[*volume.Spec]bool)
+	deviceNamesByNode := make(map[types.NodeName][]string)
+
+	for nodeName, volumeSpecs := range volumesByNode {
+		deviceNames := []string{}
+		for _, spec := range volumeSpecs {
+			volumeSource, _, err := getVolumeSource(spec)
+			// If error is occurred, skip this volume and move to the next one
```

Comment:
That comment doesn't provide any value - it describes what the code does.
Please remove.